lib/ecto_ordered.ex

```diff
@@ -34,23 +34,16 @@ defmodule EctoOrdered do
 
   """
 
+  @max 8388607
+  @min -8388607
```

Comment:
Can you document this as being a signed MEDIUMINT? I wasn't sure what I was looking at here.

Also, is there a reason for that choice in particular?